skate-plugin/build.gradle.kts

```diff
@@ -17,7 +17,10 @@ repositories { mavenCentral() }
 
 // Configure Gradle IntelliJ Plugin
 // Read more: https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html
-intellij { plugins.add("org.intellij.plugins.markdown") }
+intellij {
+  plugins.add("org.intellij.plugins.markdown")
+  plugins.add("org.jetbrains.kotlin")
```

Comment:
This is to address `NoClassDefFoundError` error. This error appears when I run tests or plugin
  The Skate (id=com.slack.intellij.skate, path=~/slack-gradle-plugin/skate-plugin/build/idea-sandbox/plugins-test/skate-plugin, version=0.1.0) plugin Plugin 'Skate' requires plugin 'org.jetbrains.kotlin' to be installed


java.lang.NoClassDefFoundError: org/jetbrains/kotlin/idea/KotlinLanguage

Reply:
The kotlin plugin is bundled anyway though, so I'm curious what difference this makes?

Reply:
I need to still dig further into this. Tests do not seem to find bundled Kotlin.

But plugin does have Kotlin bundled as I verified by installing a plugin on Android studio. If we actually include - `plugins.add("org.jetbrains.kotlin")` in the build.gradle.kts then it will result into a classLoader issue at a runtime for a plugin so this needs to be sorted out before merging the PR.